drivers/rtc/Kconfig.am1805

```diff
@@ -0,0 +1,28 @@
+#AMBIQ AM1805 RTC
+#
+#SPDX-License-Identifier: Apache-2.0
+#Copyright (c) 2023 Linumiz
+#Author: Sri Surya  <srisurya@linumiz.com>
```

Comment:
Please clean this up a bit so it looks like
```
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2023 Linumiz
# Author: Sri Surya <srisurya@linumiz.com>
```
like the .yaml file :)

Reply:
and a very minor nit, the semicolon after `# Author:` for the .yaml file as well :)